spec/models/embedded_ansible_worker_spec.rb

```diff
@@ -0,0 +1,143 @@
+describe EmbeddedAnsibleWorker do
+  subject { FactoryGirl.create(:embedded_ansible_worker) }
+
+  context "ObjectManagement concern" do
+    let(:provider)       { FactoryGirl.create(:provider_embedded_ansible) }
+    let(:api_connection) do
+      conn = double("AnsibleAPIConnection")
```

Comment:
It's kind of strange to stub `#api` to return the connection object itself.  This could be:

```ruby
let(:api_connection) { double("AnsibleAPIConnection", :api => tower_api) }
let(:tower_api)      { double("TowerAPI") }
```